config/config.go

```diff
@@ -1086,6 +1096,47 @@ func (c *AzureSDConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return checkOverflow(c.XXX, "azure_sd_config")
 }
 
+// TritonSDConfig is the configuration for Triton based service discovery.
+type TritonSDConfig struct {
+	Account            string         `yaml:"account"`
+	Cert               string         `yaml:"cert"`
+	DnsSuffix          string         `yaml:"dns_suffix"`
+	Endpoint           string         `yaml:"endpoint"`
+	InsecureSkipVerify bool           `yaml:"insecure_skip_verify"`
```

Comment:
No, in fact, I would much rather share that configuration. I wasn't sure if that would be acceptable or not so I went ahead with my own. I'll happily remove mine and use the TLS config object. Thanks for pointing this out!